assets/dev/js/editor/elements/views/base.js

```diff
@@ -277,6 +278,34 @@ BaseElementView = BaseContainer.extend( {
 		}
 	},
 
+	updateHandlesOverlay() {
+		const $overlayList = this.$el.find( '.elementor-editor-element-settings' );
+
+		const elementStyles = window.getComputedStyle( this.el ),
+			elementType = this.$el.data( 'element_type' ),
+			overflowStyles = [ elementStyles.overflowX, elementStyles.overflowY, elementStyles.overflow ],
+			isHaveOverflow = overflowStyles.includes( 'hidden' ) || overflowStyles.includes( 'auto' ),
+			isContainer = getAllElementTypes().includes( elementType );
+
+		if ( ! $overlayList?.length || ! isContainer ) {
+			return;
+		}
+
+		$overlayList[ 0 ].style.display = 'none';
+
+		// JS Hack to force browser element rerender
+		// eslint-disable-next-line no-unused-expressions
+		$overlayList[ 0 ].offsetHeight;
```

Comment:
Not sure why wouldn't the browser reflow once setting the display to none, but if anything I think this expression might be ignored by various browsers' engine optimizations, so it should at least be wrapped with a `void` call to enforce `offsetHeight`'s evaluation

```suggestion
		void( $overlayList[ 0 ].offsetHeight );
```